media/Config/reset.css

```diff
@@ -0,0 +1,31 @@
+html {
+    box-sizing: border-box;
+    font-size: 13px;
+  }
+  
+  *,
+  *:before,
+  *:after {
+    box-sizing: inherit;
+  }
+  
+  body,
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
+  p,
+  ol,
+  ul {
+    margin: 0;
+    padding: 0;
+    font-weight: normal;
+  }
+  
+  img {
+    max-width: 100%;
+    height: auto;
+  }
+  
```

Comment:
Please fix the last line; remove the red circle 
![image](https://user-images.githubusercontent.com/4616940/133736172-cf552ee4-b075-47b6-a5e5-df513fc75e2c.png)

Reply:
okay thanks for feedback